_includes/articles/cardcoins-rbf-batching.md

```diff
@@ -0,0 +1,48 @@
+{:.post-meta}
+*by [CardCoins][]*
+
+[Replace By Fee][topic rbf] (RBF, BIP-125) and [batching][payment batching] are two important tools for any enterprises
+directly interacting with Bitcoin's mempool. Fees go up, fees go down, but
+the business must always fight for fee efficiency.
+
+Each tool, while powerful, has its own complexities and nuances. For example,
+batching customer withdrawals may save on fees for the enterprise, but will likely make
+[child pays for parent][topic cpfp] (CPFP) uneconomical for a customer who wishes to speed up the transaction.
+Similarly, RBF is useful for an enterprise who takes a fee-underbidding strategy
+(their initial transaction broadcast starts at a low fee, and is slowly bid
+upwards), but it exposes their customers to [potential confusion][rbf blog] as their
+withdrawal transaction updates in their wallet. It would also be messy for the
+customer to spend from this transaction while it remains unconfirmed, as the
+enterprise will have to account for this child spend when attempting to replace
```

Comment:
Nit: I think this is more specific about the immediate burden placed on the enterprise.

```suggestion
enterprise will have to pay for this child spend when attempting to replace
```